pkgs/development/libraries/libgpg-error/default.nix

```diff
@@ -34,6 +34,16 @@ in stdenv.mkDerivation (rec {
   '' + lib.optionalString (stdenv.hostPlatform.isAarch32 && stdenv.hostPlatform.isMusl) ''
     ln -s src/syscfg/lock-obj-pub.arm-unknown-linux-gnueabi.h src/syscfg/lock-obj-pub.arm-unknown-linux-musleabihf.h
     ln -s src/syscfg/lock-obj-pub.arm-unknown-linux-gnueabi.h src/syscfg/lock-obj-pub.linux-musleabihf.h
+  ''
+  # This file was accidentally excluded from the sdist until
+  # 013720333c6ec1d38791689bc49ba039d98e16b3, post release.
+  # TODO make unconditional next mass rebuild
+  + lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
+    cp ${fetchurl {
+      url = "https://raw.githubusercontent.com/gpg/libgpg-error/50e62b36ea01ed25d12c443088b85d4f41a2b3e1/src/gen-lock-obj.sh";
```

Comment:
Note that this is the revision ID of the 1.38 version tag it seems like

 * https://github.com/gpg/libgpg-error/commit/50e62b36ea01ed25d12c443088b85d4f41a2b3e1

Not an arbitrary spooky revision ID from a third-party tree, but from their own tree it looks like.

Reply:
Yeah upstream link is https://dev.gnupg.org/source/libgpg-error/browse/master/src/gpg-error-config-test.sh.in;50e62b36ea01ed25d12c443088b85d4f41a2b3e1, but when you got to get the raw link phabricator gives something lousy-looking that doesn't contain the commit hash. That just sounds to me like something that's going to rot.

Reply:
Oh, that's alright, I was mostly pointing it out for other reviewers, showing that at least one person verified the provenance of that file :).

Reply:
Thanks, appreciated :-)